server/server_test.go

```diff
@@ -1449,7 +1449,7 @@ func TestQueryWatchMigrateInvalidate(t *testing.T) {
 		t.Fatalf("Expected -200 but got: %v", stream[0].Result[0].Bindings["a"])
 	}
 
-	expMsg := "watch invalidated: 1 error occurred: 1:3: rego_type_error: plus: invalid argument(s)\n\thave: (string, any, ???)\n\twant: (number, number, number)"
+	expMsg := "watch invalidated: 1 error occurred: 1:3: rego_type_error: plus: invalid argument(s)\n\thave: (string, any<...>, ???)\n\twant: (number, number, number)"
```

Comment:
Not sure where to leave this comment but this test case jumped out at me and made me wonder about the implementation. In this case we have an argument of type `any`, i.e., it's totally dynamic--OPA cannot tell the type of the value at compile time. `any<...>` implies the type is dynamic, but that OPA was able to enumerate the value types at compile time. This points to an issue in the eliding implementation.